tests/cephadm/test_admin_add_labels.py

```diff
@@ -0,0 +1,78 @@
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def adding_label(instance):
+    """
+    Method add multiple labels in the node
+    Args:
+        instance: cluster instance
+        node: cluster node
+    Returns:
+        boolean
+    """
+
+    node = instance.cluster.get_nodes()[0].hostname
+    out, _ = instance.installer.exec_command(
+        cmd="cephadm shell ceph orch host add "
+        + node
+        + " "
+        + node
+        + " node-exporter crash osd installer grafana prometheus alertmanager",
+        sudo=True,
+    )
+    if "Added host" in out:
+        log.info("Successfully added multiple roles in same admin node ")
+        return 0
```

Comment:
Instead of returning value I suggest to raise exception in case it fails. It will avoid using code - https://github.com/red-hat-storage/cephci/pull/1714/files#diff-7ab9de0c21c20c44d89b4a06d3e6c26c5dc9675a404aec4853228d58dfa811ccR59

Reply:
Thanks @vamahaja
Updated